containers/README.md

```diff
@@ -0,0 +1,48 @@
+Subscription Manager & Containers
+=================================
+
+This directory contains scripts and configuration files that are necessary for running
+subscription-manager in a containerized environment. This environment could be created
+using podman. Why do we care about running subscription-manager in a containerized
+environment? We believe that it could be useful for testing of subscription-manager.
+We also support running and testing of subscription-manager using vagrant, but running
+subscription-manager in VM could be a bit cumbersome. Running subscription-manager
+using podman could be more flexible for developers too.
+
+Requirements
+------------
+
+To create and use a containerized environment it is necessary to have following packages
+installed in the system:
+
+ * podman
+ * toolbox
+
+   $ sudo dnf install -y podman toolbox
+
+Create a containerized environment
+----------------------------------
+
+To create a containerized environment you can simply run following script:
+
+    $ toolbox create -i quay.io/candlepin/subscription-manager:main -c rhsm-dev
+    $ toolbox enter rhsm-dev
```

Comment:
I tried this and get following error. I also used `--verbose` to see some details:

```
[jhnidek@thinkpad-t580 subscription-manager]$ toolbox --verbose create -i quay.io/candlepin/subscription-manager:main -c rhsm-dev
DEBU Running as real user ID 1000                 
DEBU Resolved absolute path to the executable as /usr/bin/toolbox 
DEBU Running on a cgroups v2 host                 
DEBU Checking if /etc/subgid and /etc/subuid have entries for user jhnidek 
DEBU Validating sub-ID file /etc/subuid           
DEBU Validating sub-ID file /etc/subgid           
DEBU TOOLBOX_PATH is /usr/bin/toolbox             
DEBU Migrating to newer Podman                    
DEBU Toolbox config directory is /home/jhnidek/.config/toolbox 
DEBU Current Podman version is 3.2.3              
DEBU Creating runtime directory /run/user/1000/toolbox 
DEBU Old Podman version is 3.2.3                  
DEBU Migration not needed: Podman version 3.2.3 is unchanged 
DEBU Resolving container and image names          
DEBU Container: 'rhsm-dev'                        
DEBU Distribution: ''                             
DEBU Image: 'quay.io/candlepin/subscription-manager:main' 
DEBU Release: ''                                  
DEBU Resolved container and image names           
DEBU Container: 'rhsm-dev'                        
DEBU Image: 'quay.io/candlepin/subscription-manager:main' 
DEBU Release: 'main'                              
DEBU Checking if container rhsm-dev already exists 
DEBU Looking for image quay.io/candlepin/subscription-manager:main 
DEBU Looking for image quay.io/candlepin/subscription-manager:main 
Image required to create toolbox container.
Download quay.io/candlepin/subscription-manager:main (500MB)? [y/N]: y
DEBU Pulling image quay.io/candlepin/subscription-manager:main 
Trying to pull quay.io/candlepin/subscription-manager:main...
Error: Error initializing source docker://quay.io/candlepin/subscription-manager:main: Error reading manifest main in quay.io/candlepin/subscription-manager: manifest unknown: manifest unknown
Error: failed to pull image quay.io/candlepin/subscription-manager:main
```

What can be wrong?

I worked some time ago as you can see in the list of images and containers:

```
[jhnidek@thinkpad-t580 subscription-manager]$ toolbox list
IMAGE ID      IMAGE NAME                                                               CREATED
393dc0352639  quay.io/candlepin/subscription-manager:csnyder_containerized_everything  2 months ago
daf770cfb9ee  registry.fedoraproject.org/fedora-toolbox:34                             2 weeks ago

CONTAINER ID  CONTAINER NAME     CREATED         STATUS      IMAGE NAME
5182edf38c05  fedora-toolbox-34  36 minutes ago  configured  registry.fedoraproject.org/fedora-toolbox:34
dad35270e016  rhsm               2 months ago    exited      quay.io/candlepin/subscription-manager:csnyder_containerized_everything
```

Reply:
I was able to create fedora34 container using `toolbox create` this morning. Thus it seems that there is something wrong with our image on quay.io.

Reply:
Yea there is no tag pushed for "main" yet. That is probably why

Reply:
Try again with the PR review instructions please! That *should* work at this point but it's possible something is messed up.

The tag jenkins created by looking at this pr is "PR-2758" (which didn't exist until jenkins ran to jenkinsfile in this PR)

Reply:
Yeah, it makes sense. I can confirm that it works for current PR, when I use:

```
toolbox --verbose create -i quay.io/candlepin/subscription-manager:PR-2758 -c rhsm-dev
```

Thanks